actionpack/test/dispatch/content_security_policy_test.rb

```diff
@@ -23,145 +23,197 @@ def test_dup
     assert_equal copied.build, @policy.build
   end
 
+  def test_deprecated_falsy_argument
+    ActiveSupport::Deprecation.silence do
+      # want to make sure this makes a deprecation but don't know how
+
+      # this call used to empty the value but now it's just a reader
+      @policy.script_src :data
+      assert_not_nil @policy.script_src
+      assert_not_nil @policy.script_src
+
+      # Call with false will still empty the set
+      @policy.script_src(false)
+      assert_nil @policy.script_src
+    end
+  end
+
   def test_mappings
-    @policy.script_src :data
+    @policy.script_src :data, clear: true
```

Comment:
Should `clear: true` be removed here, as there also is a valid value (`:data`) passed?
```suggestion
    @policy.script_src :data
```
Same thing applies to other examples which also pass a valid argument.

Reply:
If we dont pass `clear: true` we would add more directives to script-src. It's not nesseccary on the first call but I would like to have it for the other calls since I dont want to have other data in the directive.